keps/sig-scheduling/5007-device-attach-before-pod-scheduled/README.md

```diff
@@ -786,6 +770,12 @@ enhancement:
   cluster required to make on upgrade, in order to make use of the enhancement?
 -->
 
+This feature exposes new fields such as `BindingConditions` in the
+`ResourceClaim` and `ResourceSlice`, the fields willeither be present or not.
+
+This feature uses the DRA interface and will follow the DRA upgrade/downgrade
```

Comment:
There is no "DRA upgrade/downgrade". I would just say that the fields follow standard K8s feature flag semantics for managing field data and how components interact with those fields.